pkg/storage/image.go

```diff
@@ -597,16 +598,47 @@ func GetImageService(ctx context.Context, store storage.Store, defaultTransport
 		seenRegistries[r] = true
 	}
 
+	// support global shared /etc/containers/registries.conf
+	allSystemRegistries, err := sysregistriesv2.GetRegistries(sc)
+	if err != nil {
+		return nil, err
+	}
+	insecureSystemRegistries := []string{}
+	for _, r := range allSystemRegistries {
+		if r.Insecure {
+			insecureSystemRegistries = append(insecureSystemRegistries, r.URL)
+		}
+	}
+	systemRegistries := []string{}
+	for _, r := range allSystemRegistries {
+		if r.Search {
```

Comment:
This should use `FindUnqualifiedSearchRegistries`.

Reply:
can you elaborate?